Aptoide/appcoins-wallet-android
# [hotfix] Integer overflow fix

**What does this PR do?**

Fixes integer overflow bug that happens in some currencies

**Database changed?**

   Yes | No

**Where should the reviewer start?**

ExternalBillingSerializer

**How should this be manually tested?**

-Test on Shadow Legends
-Use charles and add country=IR to the products endpoint (getPackages())
-Check if the Gem Bundles don't cost the integer overflow number (2147...)

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-XXXX](https://aptoide.atlassian.net/browse/MOB-<Jira ticket number>)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.)




**Code Review Checklist**
